packages/core/src/modules/credentials/services/CredentialService.ts

```diff
@@ -247,18 +247,20 @@ export abstract class CredentialService {
     return this.credentialRepository.findById(connectionId)
   }
 
-  public async deleteById(credentialId: string, options?: DeleteCredentialOptions): Promise<void> {
-    const credentialRecord = await this.getById(credentialId)
-
+  public async deleteById(
+    credentialRecord: CredentialExchangeRecord,
```

Comment:
It doesn't make sense to call this `deleteById` anymore, as we're deleting it by record.

```suggestion
  public async delete(
    credentialRecord: CredentialExchangeRecord,
```